accepted/calc-functions.md

```diff
@@ -249,8 +248,10 @@ Let `n1` and `n2` be two numbers. To determine `n1 % n2`:
 This algorithm takes a [`FunctionCall`] `call` whose name is a plain identifier
 and returns a number or a calculation.
 
-* If `call`'s `ArgumentInvocation` contains one or more `KeywordArg`s or one or
-  more `RestArg`s, throw an error.
+[`FunctionCall`]: ../spec/functions.md#functioncall
+
+* If `call`'s `ArgumentInvocation` contains one or more `KeywordArgument`s or
+  one or more `RestArgument`s, throw an error.
```

Comment:
my only nitpick is that this refers to `$foo...` as a `RestArgumant` where other places refer to this as an "argument list"

https://sass-lang.com/documentation/at-rules/mixin/#taking-arbitrary-arguments

That said, I did find a couple other places in the spec mentioning "rest arguments", so if this is OK then LGTM :)

Reply:
The argument list is a type of value passed to the function. The RestArgument is about describing the structure of the ArgumentInvocation